contracts/test/v0.8/ValidatorProxy.test.js

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable */
```

Comment:
Probably just some js n00bness but if I make these imports, then the linter warning goes away but then the tests dont run "not part of a module" error ?

Reply:
This is down to the fact we use 2 separate linters: eslint and prettier.

For now, in the root dir, add contracts/ to the .eslintignore file instead of this line.

This should be fixed once we convert these to typescript tests in a separate ticket.